web/skins/classic/views/console.php

```diff
@@ -231,7 +231,8 @@
 <?php
 ob_start();
 ?>
-	  <div class="container-fluid table-responsive-sm pt-2" id="monitorList">
+    <div class="pt-2"></div>
```

Comment:
Another option is to set the class "pb-2" for "id="contentButtons"", but you need to look at what this will affect.
I did the most painless option (so that nothing would break) for correct operation.
There is still a lot of work on the templates.

Reply:
I made changes.
Nothing has broken globally.